src/openrct2/scenario/Scenario.cpp

```diff
@@ -476,6 +477,22 @@ static int32_t scenario_create_ducks()
     return 1;
 }
 
+std::pair<uint32_t, uint32_t> scenario_rand_seed()
```

Comment:
Perhaps declare a struct for the prng state and return a const reference. Also I'm not a fan of the stringstream usage few lines below.

Reply:
You'd rather use no tuples at all I suppose? I like that the pair allows access like `.second` and tying with `std::tie`.

Reply:
I mean the state is essentially uint32_t[2] so why not use POD when you can?